lib/src/widgets/node_editor.dart

```diff
@@ -545,6 +547,44 @@ class _NodeEditorDataLayerState extends State<_NodeEditorDataLayer>
     }
   }
 
+  // Keyboard event handlers
+  void _onKeyEvent(KeyEvent event) {
+    final LogicalKeyboardKey key = event.logicalKey;
+    final bool metaPressed = HardwareKeyboard.instance.isMetaPressed;
```

Comment:
@WilliamKarolDiCioccio if the problem was that you couldn't get shortcuts working, it seems like it's because this checks for the meta key but not the control key, so it will only work with the `Cmd` key on a mac's keyboard (and the `Win` key on other keyboards), but not with the `Ctrl` keys.

I don't have a mac to test, but i'm pretty sure this would make it work for all systems (with a hardware keyboard)
```suggestion
    final bool metaPressed = HardwareKeyboard.instance.isMetaPressed || HardwareKeyboard.instance.isControlPressed;
```

(though you should probably rename that variable now)